From 1ff229c8e02bdd3c859d581787636cfdf674eec1 Mon Sep 17 00:00:00 2001
From: Gowtham Suresh Kumar <gowtham.sureshkumar@arm.com>
Date: Wed, 17 Nov 2021 15:28:06 +0000
Subject: [PATCH 12/24] corstone1000: Update FFA shared buffer address

FFA shared buffer address changed to 0x02000000.

The existing address 0x023F8000 is currently being used by
Optee so the virtual address returned to the SMM gateway is 0x0000.
So the buffer is moved to 0x02000000.

Signed-off-by: Gowtham Suresh Kumar <gowtham.sureshkumar@arm.com>
%% original patch: 0025-Update-FFA-shared-buffer-address.patch

%% original patch: 0025-Update-FFA-shared-buffer-address.patch

Signed-off-by: Rui Miguel Silva <rui.silva@linaro.org>
---
 include/configs/corstone1000.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/corstone1000.h b/include/configs/corstone1000.h
index a400cdef69d0..db0f91335cef 100644
--- a/include/configs/corstone1000.h
+++ b/include/configs/corstone1000.h
@@ -45,7 +45,7 @@
  * shared buffer physical address used for communication between
  * u-boot and the MM SP
  */
-#define FFA_SHARED_MM_BUFFER_ADDR	(0x023F8000)
+#define FFA_SHARED_MM_BUFFER_ADDR	(0x02000000)
 
 #define CONFIG_SYS_INIT_SP_ADDR		(CONFIG_SYS_SDRAM_BASE + 0x03f00000)
 #define CONFIG_SKIP_LOWLEVEL_INIT
-- 
2.37.1

